.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 90,
+  "arrowParens": "always",
+  "semi": true,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "jsxSingleQuote": true
```

Comment:
Lots of :heart: here!
Just a question, can we add `trailingCommas: none`? What are your feelings about trailing commas?

Reply:
I prefer to set `trailingCommas` to `none` as well (more like a personal preference)